student_frame.rb

```diff
@@ -0,0 +1,12 @@
+require './person_frame'
+
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
```

Comment:
- [ ] In **super()** kindly rearrange the argumens because in the parent class **age** comes brfore the **name** :thinking: So your super will look something like below :arrow_down:

`super(age, name, parent_permission)`

Reply:
Change is made.